SPECS/grub2/grub2.spec

```diff
@@ -327,6 +328,9 @@ cp $GRUB_PXE_MODULE_SOURCE $EFI_BOOT_DIR/$GRUB_PXE_MODULE_NAME
 %endif
 
 %changelog
+* Fri Aug 11 2023 Cameron Baird <cameronbaird@microsoft.com> - 2.06-11
+- Enable support for grub2-mkconfig grub.cfg generation
```

Comment:
Change the entry to indicate the change to `/etc/default/grub`'s config properties